### ISSUE TITLE
Run VSCode tasks in a single session

### ISSUE THREAD

**Issue by pt12lol** [microsoft/vscode#78319]
I would like to migrate my local deployment commands to VSCode task. I need to run them as root. When I did those in the terminal, I was asked to provide the password only once.

```
$ sudo echo hello                                                                                                                                                                                                                                                        
[sudo] password for pt12lol: 
hello
$ sudo echo hello                                                                                                                                                                                                                                                        
hello
```

Unfortunately when I configured my VSCode task:

```
{
    "version": "2.0.0",
    "tasks": [
        {
            "label": "Hello task",
            "type": "shell",
            "command": "sudo echo hello"
        }
    ]
}
```

I suppose it runs every task in a separate session, and I am asked to provide my password every single time.

```
> Executing task: sudo echo hello <

[sudo] password for pt12lol: 
hello

Terminal will be reused by tasks, press any key to close it.

> Executing task: sudo echo hello <

[sudo] password for pt12lol: 
hello

Terminal will be reused by tasks, press any key to close it.
```

I would like to set up tasks deployment so that it keeps last sessions and I am not asked for my password every single time.

**Comment by alexr00** [comment]
duplicate of https://github.com/microsoft/vscode/issues/44152
